daisy_workflows/linux_common/utils/diskutils.py

```diff
@@ -37,7 +39,20 @@ def log_key_value(key, value):
   print(_STATUS_PREFIX + "<serial-output key:'%s' value:'%s'>" % (key, value))
 
 
-def MountDisk(disk) -> guestfs.GuestFS:
+def get_physical_drives():
```

Comment:
@MahmoudNada0, in the case of a failure your method won't be consistent in the sense of the returned value:

1. If `lsblk` is completed successfully then you will return all disks including the bootable one;
2. If `lsblk` fails then you will return only a data disk.

As a caller I will never know if the output contains a bootable disk or not because I will never know if an error has happen in `lsblk`.

However, I would agree with @EricEdens here, that if `lsblk` fails then it is not something we should hide, I would fail with it.

Reply:
Ok Good, Then we can instead of returning only the "/dev/sdb" I'll return an error and make the import fail

Reply:
As agreed with @yswe we will warn that failed to execute 'lsblk' cmd, and continue anyway assuming that there are two disks only ['/dev/sda', '/dev/sdb']